tests/Microsoft.Identity.Test.Integration.netfx/HeadlessTests/RuntimeBrokerTests.cs

```diff
@@ -265,6 +264,70 @@ public async Task WamAddDefaultScopesWhenNoScopesArePassedAsync(string scopes)
 
             Assert.IsTrue(!string.IsNullOrEmpty(ex.ErrorCode));
         }
+
+        [RunOn(TargetFrameworks.NetStandard | TargetFrameworks.NetCore)]
+        public async Task WamUsernamePasswordPopTokenEnforcedWithCaOnValidResourceAsync()
+        {
+            //Arrange
+            var labResponse = await LabUserHelper.GetDefaultUserAsync().ConfigureAwait(false);
+            
+            string popUser = "popUser@msidlab4.onmicrosoft.com";
+
+            string[] scopes = { "https://msidlab4.sharepoint.com/user.read" };
```

Comment:
Are you able to actually make a web request to the actual SPO ? Am curios for: 

1. Does SPO issue server nonce?
2. Does SPO accept a POP token without a server nonce?